actions/index.js

```diff
@@ -47,38 +48,72 @@ export const startScan = () => {
       }
     }, true);
   };
-}
+};
+
+//on android device, we should ask permission
+const requestLocationPermission = async () => {
+  try {
+    const granted = await PermissionsAndroid.request(
+      PermissionsAndroid.PERMISSIONS.ACCESS_COARSE_LOCATION,
+      {
+        title: 'Location permission for bluetooth scanning',
+        message: 'wahtever',
```

Comment:
There is a minor spelling error here, I can fix if you'd like

Reply:
If you don't mind to fix it, no problem for me. Thanks for the answer.